babel.config.js

```diff
@@ -0,0 +1,30 @@
+// can't be `.babelrc` because Jest won't work with it: https://github.com/facebook/jest/issues/10256
+
+module.exports = {
+  sourceMaps: true,
+  presets: [
+    '@babel/env',
+    [
+      '@babel/typescript',
+      {
+        jsxPragma: 'h',
+      },
+    ],
+  ],
+  plugins: [
+    [
+      '@babel/plugin-transform-react-jsx',
+      {
+        pragma: 'h',
+      },
+    ],
+    '@babel/plugin-syntax-dynamic-import',
+    '@babel/plugin-syntax-import-meta',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-proposal-json-strings',
+    '@babel/plugin-proposal-function-sent',
+    '@babel/plugin-proposal-numeric-separator',
+    '@babel/plugin-proposal-throw-expressions',
+    '@babel/plugin-proposal-export-default-from',
```

Comment:
The following plugins are already part of the `@babel/env` package and can be removed:
```
'@babel/plugin-syntax-dynamic-import',
'@babel/plugin-syntax-import-meta', 
'@babel/plugin-proposal-class-properties', 
'@babel/plugin-proposal-json-strings', 
'@babel/plugin-proposal-numeric-separator', 
```

Reply:
Thanks @DannyvanderJagt , very well spotted, I have cleaned up all but the '@babel/plugin-proposal-class-properties', seems to be depending on that. Tests are failing when removed